File: app/src/main/java/com/example/recyclerview/remote/retrofit/SOService.java
package com.example.recyclerview.remote.retrofit;
import com.example.recyclerview.model.ApiGitHub;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

//quan ly viec truy xuat voi server
public interface SOService {
 
   @GET("users")
   Call<List<ApiGitHub>> EXAMPLE_CALL();

}

File: app/src/main/java/com/example/recyclerview/remote/retrofit/SOService.kt
package com.example.recyclerview.remote.retrofit

import com.example.recyclerview.model.ApiGitHub

import retrofit2.Call
import retrofit2.http.GET

//quan ly viec truy xuat voi server
interface SOService {

    @GET("users")
    fun EXAMPLE_CALL(): Call<List<ApiGitHub>>

}